rpc/src/main.rs

```diff
@@ -24,7 +24,7 @@ async fn main() -> Result<()> {
         Command::Request(req) => match req {
             Request::CheckNullifiers { nullifiers } => {
                 let host_port =
-                    format!("http://{}:{}", config.rpc.host_port.host, config.rpc.host_port.port);
+                    format!("http://{}:{}", config.rpc.endpoint.host, config.rpc.endpoint.port);
```

Comment:
Should we rename the variable here as well from `host_port` to something else?

Reply:
done